plugins/GSdx/GSdx.rc

```diff
@@ -199,42 +200,42 @@ BEGIN
-    CONTROL         "",IDC_OSD_OPACITY_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,107,100,76,15
-    RTEXT           "100",IDC_OSD_OPACITY_AMOUNT,184,104,11,8
-    // Log Settings
-    GROUPBOX        "Log Messages",IDC_STATIC,7,125,196,52,0,WS_EX_TRANSPARENT
-    LTEXT           "Timeout (seconds):",IDC_OSD_TIMEOUT_TEXT,16,139,75,8
-    EDITTEXT        IDC_OSD_TIMEOUT_EDIT,112,135,83,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
-    CONTROL         "", IDC_OSD_TIMEOUT,"msctls_updown32", UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
-    LTEXT           "Max:",IDC_OSD_MAX_LOG_TEXT,16,158,25,8
-    EDITTEXT        IDC_OSD_MAX_LOG_EDIT,112,155,83,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
-    CONTROL         "", IDC_OSD_MAX_LOG,"msctls_updown32", UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
-    // OK/CANCEL
-    DEFPUSHBUTTON   "OK",IDOK,52,183,50,14
-    DEFPUSHBUTTON   "Cancel",IDCANCEL,108,183,50,14
+    LTEXT           "Red:",IDC_OSD_COLOR_RED_TEXT,15,84,25,8
+    CONTROL         "",IDC_OSD_COLOR_RED_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,106,82,76,15
+    RTEXT           "255",IDC_OSD_COLOR_RED_AMOUNT,183,85,11,8
+    LTEXT           "Green:",IDC_OSD_COLOR_GREEN_TEXT,15,101,25,8
+    CONTROL         "",IDC_OSD_COLOR_GREEN_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,106,99,76,15
+    RTEXT           "255",IDC_OSD_COLOR_GREEN_AMOUNT,183,102,11,8
+    LTEXT           "Blue:",IDC_OSD_COLOR_BLUE_TEXT,15,119,25,8
+    CONTROL         "",IDC_OSD_COLOR_BLUE_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,106,116,76,15
+    RTEXT           "255",IDC_OSD_COLOR_BLUE_AMOUNT,183,119,11,8
+    LTEXT           "Opacity:",IDC_OSD_OPACITY_TEXT,15,137,50,8
+    CONTROL         "",IDC_OSD_OPACITY_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,106,133,76,15
+    RTEXT           "100",IDC_OSD_OPACITY_AMOUNT,183,137,11,8
+    GROUPBOX        "Log Messages",IDC_STATIC,6,154,196,52,0,WS_EX_TRANSPARENT
+    LTEXT           "Timeout (seconds):",IDC_OSD_TIMEOUT_TEXT,15,168,75,8
+    EDITTEXT        IDC_OSD_TIMEOUT_EDIT,111,164,83,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_OSD_TIMEOUT,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
+    LTEXT           "Max:",IDC_OSD_MAX_LOG_TEXT,15,187,25,8
+    EDITTEXT        IDC_OSD_MAX_LOG_EDIT,111,184,83,14,ES_RIGHT | ES_AUTOHSCROLL | ES_NUMBER
+    CONTROL         "",IDC_OSD_MAX_LOG,"msctls_updown32",UDS_SETBUDDYINT | UDS_ALIGNRIGHT | UDS_AUTOBUDDY | UDS_ARROWKEYS | UDS_NOTHOUSANDS,0,0,11,14
+    DEFPUSHBUTTON   "OK",IDOK,52,213,50,14
+    DEFPUSHBUTTON   "Cancel",IDCANCEL,108,213,50,14
+    LTEXT           "Position",IDC_OSD_MONPOS_TEXT,15,35,28,8
+    COMBOBOX        IDC_OSD_MONITOR_POS,111,33,83,30,CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
```

Comment:
Maybe increase the height of the drop list so it shows the entire list properly in gsdumpgui.